### PR TITLE
Fix editing of json values

### PR DESCRIPTION
**Pull request by hwbllmnn**
@terrestris/devs Please review.

### CODE REVIEW

**Review by annarieger — APPROVED** [review]

**Comment by annarieger** [comment]
Very nice! 🍍

File: src/Component/FormField/JSONEditor/JSONEditor.tsx [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
 
 import './JSONEditor.less';
 
 type OwnProps = {
   value?: string;
-  onChange?: (value: string) =>  void;
+  onChange?: (value: string) => void;
 };
 
-export type JSONEditorProps = OwnProps & Omit<CodeMirrorProps, 'onBeforeChange' | 'value' | 'onChange'>;
```
Comment (dnlkoch) [inline src/Component/FormField/JSONEditor/JSONEditor.tsx:17]: Non blocking, but we should allow passing the `EditorProps` to the wrapped component.

**Review by dnlkoch — APPROVED** [review]
Thanks for this!

File: src/Component/FormField/JSONEditor/JSONEditor.tsx (hunk 1 above)
Comment (hwbllmnn) [reply src/Component/FormField/JSONEditor/JSONEditor.tsx:17]: Done in f52be66

File: package.json [hunk 2]
```diff
@@ -30,18 +30,17 @@
   "homepage": "https://github.com/terrestris/shogun-admin#readme",
   "dependencies": {
     "@ant-design/icons": "^4.3.0",
+    "@monaco-editor/react": "^4.0.11",
```
Comment (KaiVolland) [inline package.json:33]: :star_struck:

**Pull request merged**